packages/manager/modules/bm-server-components/src/general-information/inputs/template.html

```diff
@@ -0,0 +1,112 @@
+<!-- dynamic inputs form depending on the OS template -->
```

Comment:
Some formatting would be nice on this file, this should have been done with the pre-commit hook from husky.